NewDawn/NewDawn/ViewControllers/RegistrationViewControllers/ProfileGNBViewController.swift

```diff
@@ -51,6 +52,7 @@ class ProfileGNBViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        Mixpanel.mainInstance().time(event: "Registration Duration")
```

Comment:
Let's create a constants file to store this identifier since it is re-used in two places